.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: bash
+sudo: required
```

Comment:
Why? We're not doing anything that requires `sudo` and using the container infra instead makes our builds happen sooner.